app/api/api_v1/routers/search.py

```diff
@@ -58,8 +58,12 @@ def _search_request(db: Session, search_body: SearchRequestBody) -> SearchRespon
     is_browse_request = not search_body.query_string
     if is_browse_request:
         search_body.all_results = True
-        search_body.documents_only = True
         search_body.exact_match = False
+
+        if search_body.concept_filters:
```

Comment:
As text is now not the only field searched on in the passages index in vespa we have to allow searching on the passages index even when the query_string is None.

Reply:
I see. So this is to handle queries that are just searching for concepts? In its current form it looks like this would be quite tricky to maintain. Perhaps we could have two separate checks for each scenario, rather than a nested one? Perhaps checking first for "concept search", then an elif for browse search. What do you think?

Reply:
Appreciate its a bit muddy already. I'd much rather have an explicit flag for browse searches, rather than inferring them from empty strings. But probably not the time to address that